supabase/functions/stats/index.ts

```diff
@@ -13,9 +14,54 @@ const failActions = [
   'update_fail',
   'download_fail',
 ]
+
+// ios sends 13 fields while android sends 11 fields
+const jsonRequestSchema = z.object({
+  app_id: z.string({
+    required_error: "App ID is required",
+    invalid_type_error: "App ID name must be a string",
+  }),
+  device_id: z.string({
+    required_error: "Device ID is required",
+    invalid_type_error: "Device ID must be a string",
+  }).max(36),
+  platform: z.string({
```

Comment:
A platform could a union perhaps?

Reply:
When I checked now, nope since it's used as a container for mixing types (e.g string and number).